### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by edenlum**
## Checklist before requesting a review:
- [x] I set the appropriate labels on the pull request.
- [x] I have added/updated the release note draft (if necessary).
- [x] I have updated the documentation to reflect my changes (if necessary).
- [x] All function and files are well documented. 
- [x] All function and classes have type hints.
- [x] There is a licenses in all file.
- [x] The function and variable names are informative. 
- [x] I have checked for code duplications.
- [x] I have added new unittest (if necessary).

### CODE REVIEW

File: tutorials/quick_start/README.md [hunk 1]
```diff
@@ -15,6 +15,159 @@ in the future.
 - ultralytics: [https://ultralytics.com](https://ultralytics.com)
 - keras-applications: [https://keras.io/api/applications](https://keras.io/api/applications/)
```
Comment (Idan-BenAmi) [inline tutorials/quick_start/README.md:16]: 1. Move the "Supported libraries" section below the table.
2. Add new section "Supported Features" which includes the following:
 - Quantize pretrained models from various model libraries [link to "Supported libraries] by one click
 - Evaluate the performance of the floating point model and the quantized model.
 - Use MCT's Post-Training Quantization (PTQ) scheme [link to example]
 - Use MCT's advanced quantization techniques (such as GPTQ and Mixed Precision) [link to the example]
3. Add section title: "Results" before the table.

"" (

File: tutorials/quick_start/README.md [hunk 2]
```diff
@@ -15,6 +15,159 @@ in the future.
...
 
+<table>
```
Comment (Idan-BenAmi) [inline tutorials/quick_start/README.md:18]: Let's remove the link below mobilenet_v2 for now, and push this modification without the notebook.

**Review by Idan-BenAmi — CHANGES_REQUESTED** [review]

**Comment by Idan-BenAmi** [comment]
I would locate the notebook file under notebooks folder with informative name.

File: tutorials/quick_start/README.md (hunk 1 above)
Comment (edenlum) [reply tutorials/quick_start/README.md:16]: do we have an example of gptq in quick start?

Comment (Idan-BenAmi) [reply tutorials/quick_start/README.md:16]: Scroll down this page, you will see: "Advanced model quantization example"

File: tutorials/notebooks/example_quick_start_imagenet.ipynb [hunk 3]
```diff
@@ -0,0 +1,323 @@
+{
```
Comment (Idan-BenAmi) [inline tutorials/notebooks/example_quick_start_imagenet.ipynb:1]: 1. You need to clear the output before commit.
2. I think better to rename it to "example_quick_start_torchvision.ipynb"
3. check that you add a license at the end of the file. (you can copy from other notebooks)

File: tutorials/notebooks/example_quick_start_torchvision.ipynb [hunk 4]
```diff
@@ -0,0 +1,199 @@
...
+      "metadata": {
+        "collapsed": false,
+        "id": "c9e7b10d2bfe67d4"
+      },
+      "source": [
+        "# Quick Start\n",
+        "\n",
+        "[Run this tutorial in Google Colab](https://colab.research.google.com/github/sony/model_optimization/blob/quickstart-table/tutorials/notebooks/example_quick_start_torchvision.ipynb)\n",
+        "\n",
+        "Steps:\n",
```
Comment (Idan-BenAmi) [inline tutorials/notebooks/example_quick_start_torchvision.ipynb:15]: Expand this section a little bit:

Overview
This tutorial shows how to use "quick-start" with a pre-trained model from the torchvision library.
The following steps will be covered:
...the steps you mentioned...

File: tutorials/notebooks/example_quick_start_torchvision.ipynb [hunk 5]
```diff
@@ -0,0 +1,199 @@
...
+      "cell_type": "markdown",
+      "id": "d0e81b09e6d30873",
+      "metadata": {
+        "collapsed": false,
+        "id": "d0e81b09e6d30873"
+      },
+      "source": [
+        "## Setup\n",
+        "\n",
+        "install and import relevant packages"
```
Comment (Idan-BenAmi) [inline tutorials/notebooks/example_quick_start_torchvision.ipynb:33]: install --> Install

**Review by Idan-BenAmi — APPROVED** [review]
